decidim-core/app/helpers/decidim/sanitize_helper.rb

```diff
@@ -112,12 +112,12 @@ def add_line_feeds(text)
       add_line_feeds_to_paragraphs(add_line_feeds_to_list_items(text))
     end
 
-    def content_handle_locale(body, all_locales, extras, links, strip_tags)
+    def content_handle_locale(body, all_locales, links, strip_tags)
       handle_locales(body, all_locales) do |content|
         content = strip_tags(sanitize_text(content)) if strip_tags
 
         renderer = Decidim::ContentRenderers::BlobRenderer.new(content)
-        content = renderer.render(links:, extras:).html_safe
+        content = renderer.render.html_safe
 
         content = Decidim::ContentRenderers::LinkRenderer.new(content).render if links
```

Comment:
because of this line, we cannot remove the links parameter. 
This is being set to true , by the decidim-core/app/helpers/decidim/sanitize_helper.rb 

```
    def render_sanitized_content(resource, method, presenter_class: nil)
      content = present(resource, presenter_class:).send(method, links: true, strip_tags: !try(:safe_content?))
     # .....
    end
```

Reply:
What about in the editor_locales method? I think I asked here: https://github.com/tremend-cofe/decidim/pull/444#discussion_r2182495291

Reply:
> What about in the editor_locales method? I think I asked here: [tremend-cofe#444 (comment)](https://github.com/tremend-cofe/decidim/pull/444#discussion_r2182495291)

Yeah. I have forgot :) Fixed in c8b1605